SYCL/ESIMD/histogram_raw_send.cpp

```diff
@@ -6,7 +6,7 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: gpu
+// REQUIRES: gpu-intel-gen9, gpu
```

Comment:
is "gpu" required to stay on the list here?
Should it be gpu-intel-gen9 only?

Reply:
IMO, gpu-intel-gen9 should be enough.